checkstyle.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
I don't see checkstyle plugin in pom.xml. Without this plugin this checkstyle.xml file makes no sense